truckersmp-cli

```diff
@@ -91,6 +94,25 @@ class AppId:
     proton = {}
 
 
+def check_libsdl2():
+    """
+    Check whether SDL2 shared object file is present.
+
+    If SDL2 is detected, this function returns True.
+    Otherwise, this returns False.
+
+    Currently we check it only on Linux systems.
+    """
+    if platform.system() != "Linux":
+        return True
```

Comment:
Do you want to support other platforms in the future?

Reply:
Yes, Wine can run on several POSIX-compliant OSes.

I guess BSD OSes are the same as Linux systems but not tested. Example: [FreeBSD](https://freebsd.pkgs.org/11/freebsd-amd64/sdl2-2.0.10_1.txz.html)

Regarding macOS, I'm not sure whether SDL2 and other dependencies are bundled with "Wine for macOS" (= whether SDL2 check is needed on macOS). I would be happy if someone could test on the OS.

BTW Proton is provided only for Linux as of May 2020. Proton can be used by default only on Linux and we can use `platform` for OS checking.